src/spatial/2d.js

```diff
@@ -81,6 +82,14 @@ Crafty.c("2D", {
      * @see ._attr
      */
     _z: 0,
+
+    /**@
+     * #._globalZ
+     * @comp 2D
+     * When two entities overlap, the one with the larger `_globalZ` will be on top of the other.
+     */
+    _globalZ: null,
```

Comment:
That's true, we could move that to be triggered by Reorder.  I'll leave that out of this set of changes, though.